index.html

```diff
@@ -60,11 +60,11 @@ <h1>I’m Alexandra <br>Glad to see you!</h1>
        
         <section class="work-section">
             <div class="projects project1">
-                <img class ='project-image' src="images/project1.png" alt="">
+                <img class ='project-image' src="images/project1.png" alt="screenshot of project one">
```

Comment:
- [ ] Great job adding the alt attribute to some of your image elements. It appears that you had missed some. Kindly have a look at [this](https://github.com/Ridwanullahi-code/professional-portfolio/blob/e3f4cf83598b5b849385270861f7e750b87483f9/index.html#L49) example missed a section and add the alt attribute as a text. This way your project will be more accessible.

**Kindly add an alt attribute text to all the images in the project**